### ISSUE TITLE
Can't have multiple placeholders for kills,deaths...

### ISSUE THREAD

**Issue by Welnnys** [Ktar5/Info-Board#152]
Hey I can have multiple placeholders for kills or deaths or things like that on my scoreboard.I want to have my clansystem kills placeholder and my player kills placeholder but I can't at the same time.Please help.Thanks for reading that. <3

**Comment by pixar02** [comment]
You should be able to have multiple placeholders on the same line. `'&e%player_health_rounded%&a/&c%player_max_health_rounded%'` is an example of 2 placeholders on the same line. (in the default config)
I don't know all the placeholders out of the top of my head... 
Also don't forget to download the correct expansion that comes with those placeholders...

**Comment by Welnnys** [comment]
No, I meant i cant have the placeholders for the same thing like %clans_clan_kills% and %statistic_player_kills% .

**Comment by pixar02** [comment]
Ooh... yeay that bug... I quick fix is to add &r at the end of 1 of the 2 placeholders... 
ex: '....%clans_clan_kills%&r' and '%statistic_player_kills%'

The way scoreboards works is with every line you have a score... (scores go from 1 to 15) and since your first placeholder has the pontial to be the same as the other one it doesn't it tries to have the same "value" at a diferent score... (you placeholder is a Key that unlocks a score, but if the Key already exists it can't have 2 Keys with the same value....)

I fixed this in the recoded version of IBR...
